<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTransportItem</class>
 <widget class="QBaseWidget" name="QTransportItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTransportItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>3</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../microlib.qrc">:/images/downloadt</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSize">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPath">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>目标位置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>3</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblOperation">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>操作</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnStatus">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>3</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPix">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>3</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../microlib.qrc">:/images/downloadt</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>lblIcon</zorder>
  <zorder>lblName</zorder>
  <zorder>lblStatus</zorder>
  <zorder>lblSize</zorder>
  <zorder>lblPath</zorder>
  <zorder>btnClose</zorder>
  <zorder>btnStatus</zorder>
  <zorder>lblPix</zorder>
  <zorder>lblOperation</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QBaseWidget</class>
   <extends>QWidget</extends>
   <header>qbasewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../microlib.qrc"/>
 </resources>
 <connections/>
</ui>
